bulletin_board/server/db/migrate/20200826103217_change_client_fields.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class ChangeClientFields < ActiveRecord::Migration[6.0]
+class ChangeClientFields < ActiveRecord::Migration[6.1]
```

Comment:
I needed to change the migration defaults to Rails 6.1 to avoid schema changes.

If we keep it as 6.0, the primary keys in the `db/schema.rb` would be changed from `:bigint` to `:integer`.